Added SC storage recommendations (#4938)
Particular/docs.particular.net: servicecontrol/servicecontrol-instances/hardware.md

Real disk, CPU, RAM, and network performance can be monitored with the Windows Resource Monitor and/or Windows Performance counters.

### Storage

It is recommended to:

> - Store ServiceControl data on its own isolated disk. This makes low-level resource monitoring easy and ensures different applications are not competing for storage IOPS.

Reviewer: ```suggestion
- Store ServiceControl data on a dedicated disk. This makes low-level resource monitoring easy and ensures different applications are not competing for storage IOPS.
```

Reviewer 2: I like this suggestion

> - Store ServiceControl data on its own isolated disk. This makes low-level resource monitoring easy and ensures different applications are not competing for storage IOPS.

Reviewer: Or:

```suggestion
- Store ServiceControl data on an isolated disk. This makes low-level resource monitoring easy and ensures different applications are not competing for storage IOPS.
```
